common/dwr.h

```diff
@@ -51,6 +51,8 @@
 
 #define FAST_TEXT(x)              (x->flags[ 7] & 0xc0)
 #define SPEED_HACKS(x)            (x->flags[ 7] & 0x30)
+#define MEDIUM_TEXT(x)            (x->flags[15] & 0x03)
+#define DOUBLE_WALKING(x)         (x->flags[15] & 0x30)
```

Comment:
There is still some room in the array, though it is getting pretty full. I'm not really wanting to expand it until necessary since I don't expect there will be many new features soon.

```suggestion
#define MEDIUM_TEXT(x)            (x->flags[10] & 0x30)
#define DOUBLE_WALKING(x)         (x->flags[10] & 0xc0)
```